apps/rig_inbound_gateway/lib/rig_inbound_gateway/api_proxy/serializer.ex

```diff
@@ -7,6 +7,8 @@ defmodule RigInboundGateway.ApiProxy.Serializer do
   alias Plug.Conn.Query
   alias RigInboundGateway.Proxy
 
+  @typep headers :: [{String.t, String.t}, ...]
```

Comment:
What if old_headers is empty? I mean, sending a request without any headers would look more like [{String.t, String.t}]